src/app/settings/edit-email/page.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+import React, { useState } from 'react';
+import styles from './page.module.css';
+import InputField from '@/components/textbox/textbox';
+import ActiveButton from '@/components/Active-Button/ActiveButton';
+import Image from 'next/image';
+import backIcon from '@/assets/BackArrow.svg';
+
+import Link from 'next/link';
+
+const Editaccount: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [textboxentry, settextboxentry] = useState(false);
+
+  const handleEmailChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    setEmail(value);
+    if (value) {
+      settextboxentry(true);
+    } else {
+      settextboxentry(false);
+    }
+  };
```

Comment:
add new line after line 23 for readability between the return block and this function